### ISSUE TITLE
Zelda TOTK - Vertex Explosions

### ISSUE THREAD

**Issue by SoRadGaming** [yuzu-emu/yuzu#10645]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1460

### Description of Issue

Playing Zelda TOTK, I randomly get Vertex Explosions.
I was able to get it to happen twice at random when teleporting to the Depths location.
(Also never had this issue with 60 hours on earlier builds) - **To fix just reload save or teleport again**
![image](https://github.com/yuzu-emu/yuzu/assets/47128515/7b304125-5c1c-48c5-b7da-dd2b9671a97a)

### Expected Behavior

No Vertex Explosion 

### Reproduction Steps

Constantly teleport to the same tp location in forge construct in depths like 10 times, then teleport to sky island, then teleport back. Happens randomly tho.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11665980/yuzu_log.txt)

### System Configuration

CPU: 5800x3D
GPU/Driver: 5700XT 8GB - 23.5.2
RAM: 32 GB
OS: Windows 11

**Comment by Squall-Leonhart** [comment]
Some mods are only working properly on 1.1.1 and 1.1.2, and AMD cards don't work properly with forced AF levels (this is not what the AF fix corrects in the mods)

**Comment by SoRadGaming** [comment]
> Some mods are only working properly on 1.1.1 and 1.1.2, and AMD cards don't work properly with forced AF levels (this is not what the AF fix corrects in the mods)
> 
>     * disable mods
> 
>     * set AF to default in the yuzu settings.

I'll try that and close the issue if it's resolved, as it's a mod bug, not Yuzu one in that case. The main reason I opened the issue is that it's a new issue caused by an update. Also, "AMD cards don't work properly with forced AF levels", in Yuzu? Because I know there's a long-time misunderstanding that it only works in DX9, is that what you're referring to?

**Comment by Squall-Leonhart** [comment]
> Because I know there's a long-time misunderstanding that it only works in DX9, is that what you're referring to?

The Yuzu AF options are broken on AMD, this is nothing to do with AMD's support for AF or misunderstanding of the Driver CP options.

**Comment by James-Coyne** [comment]
I experience this issue occasionally after weapons in the Depths, and is usually fixed by switching between Normal and High GPU accuracy. I'm running an Nvidia card (4090).

**Comment by xTeixeira** [comment]
> > Because I know there's a long-time misunderstanding that it only works in DX9, is that what you're referring to?
> 
> The Yuzu AF options are broken on AMD, this is nothing to do with AMD's support for AF or misunderstanding of the Driver CP options.

Is that also true for Linux RADV?

**Comment by SoRadGaming** [comment]
The bug was recreated with only dynamic fps mod, playing without the mod to try and get a clean log and confirm it's not a mod issue. (It will take a while due to the hard to reproduce nature of the issue)

**Comment by Pockets-byte** [comment]
I have a similar issue- usually happens when I get close to maximum memory usage, but when memory isn't maxed out, dropping a weapon usually fixes it, fedora 36, intel iris, 16 gigs of ram

**Comment by mdindoffer** [comment]
I am also still experiencing these random vertex explosions.
RTX 4060 TI, EA 4040, Totk 1.2.0, no FPS mods, 1.5x scaling.
It usually happens in the depths, but it happened to me also on the surface. Interestingly, **saving** the game makes the issue go away.

**Comment by Desmondchaser27** [comment]
Experiencing these as well on an RTX 4090 and AMD Ryzen 7900x.  I do have mods enabled though.  But I'm at 1x scaling on Yuzu because the game itself has a mod to upscale.  But it happens in a lot of places around Hyrule.  It's mostly Depths, and those tend to be the worse (like the OP's image).  The other ones (above in Hyrule) seem to happen with fog effects every once in awhile or the particles during a teleport but only once loading has finished and Link is coming out of the teleport.  These problems in Hyrule usually go away on their own either over time or once an animation finishes (the teleporting).

But the Depths version of this issue requires manual intervention, such as teleporting out of the Depths or finagling in menus sometimes helps.  I'm hesitant to actually save the game during one of these for fear that somehow the game state will get screwed up?  But point is, I think all of these random effect explosions are all probably related to the same bug (or set of bugs?).
